sdk/communication/azure-communication-mediacomposition/swagger/README.md

```diff
@@ -0,0 +1,43 @@
+# Azure Communication Media Composition library for Java
+
+> see https://aka.ms/autorest
+## Getting Started
+
+To build the SDK for Communication Media Composition library, simply Install AutoRest and in this folder, run:
+
+### Setup
+```ps
+Fork and clone https://github.com/Azure/autorest.java
+git checkout main
+git submodule update --init --recursive
+mvn package -Dlocal
+npm install
+npm install -g autorest
+```
+
+### Generation
+
+```ps
+cd <swagger-folder>
+autorest README.md --java --v4 --use=@autorest/java@4.0.1
+```
+
+### Code generation settings
+``` yaml
+tag: package-preview-2021-10
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/85fd44e61de4bb5d4d18933ecf72ae2bcc657f45/specification/communication/data-plane/MediaComposition/preview/2021-12-31-preview/CommunicationMediaComposition.json
+add-context-parameter: true
+custom-types-subpackage: models
+models-subpackage: implementation.models
+```
+
+``` yaml
+java: true
+output-folder: ..\
+license-header: MICROSOFT_MIT_SMALL
+namespace: com.azure.communication.mediacomposition
+generate-client-as-impl: true
```

Comment:
If we don't plan to write a convenience layer manually, consider making this false and move "models-subpackage" to just "models". I wonder how that works out.

Reply:
That's a great suggestion! I'll give that a shot

Reply:
We can do that for now, but let's keep in mind that long-term down the road this will be a full blown normal SDK with convenience layer.

Reply:
Given that, I thin I'll keep as is for now to avoid further changes as we move towards a normal SDK w/ convenience layer